test/recipes/04-test_encoder_decoder.t

```diff
@@ -9,18 +9,43 @@
 use strict;
 use warnings;
 
-use OpenSSL::Test::Simple;
-use OpenSSL::Test qw/:DEFAULT srctop_file bldtop_dir/;
-use Cwd qw(abs_path);
+use OpenSSL::Test qw/:DEFAULT srctop_dir srctop_file bldtop_dir bldtop_file/;
+use OpenSSL::Test::Utils;
 
-setup("test_encoder_decoder");
+BEGIN {
+    setup("test_encoder_decoder");
+}
 
-plan tests => 1;
+use lib srctop_dir('Configurations');
+use lib bldtop_dir('.');
+use platform;
 
-$ENV{OPENSSL_MODULES} = abs_path(bldtop_dir("providers"));
-$ENV{OPENSSL_CONF} = abs_path(srctop_file("test", "default-and-legacy.cnf"));
+my $no_fips = disabled('fips') || ($ENV{NO_FIPS} // 0);
```

Comment:
Just `disabled('fips') should be sufficient.  Why the extra?

Reply:
Copied from other places that do the same.